static/src/javascripts/projects/commercial/modules/prebid/prebid.spec.js

```diff
@@ -48,7 +48,7 @@ describe('initialise', () => {
             _disableAjaxTimeout: false,
             _mediaTypePriceGranularity: {},
             _priceGranularity: 'custom',
-            _publisherDomain: 'null',
+            _publisherDomain: 'http://localhost',
```

Comment:
Why did this need to change?

Reply:
`window.location.origin` is now set to `http://localhost`, whereas in the previous version of Jest, it was set to `'null'`

I think this relates to an update to JSDOM, but I haven't been able to confirm.